core-web/src/main/java/org/silverpeas/core/web/util/viewgenerator/html/arraypanes/AbstractArrayPane.java

```diff
@@ -453,29 +379,18 @@ protected String printSortJavascriptFunction() {
     sb.append("cursor: \"move\",");
     sb.append("forcePlaceholderSize: true,");
     sb.append("helper: fixArrayPaneWidthHelper,");
-    if (StringUtil.isDefined(getUpdateSortJavascriptCallback())) {
-      sb.append("update: function(e, ui){").append(getUpdateSortJavascriptCallback()).append(";}");
-    }
     sb.append("}).disableSelection();");
     sb.append("</script>");
     return sb.toString();
   }
 
-  public String getUpdateSortJavascriptCallback() {
```

Comment:
I don't know about all signatures, but this one is used.
I think there is chance the others too...

Reply:
Huuu, it is used for move line js callback. Fuck